stable/jenkins/values.yaml

```diff
@@ -27,6 +27,12 @@ Master:
   SlaveListenerPort: 50000
   LoadBalancerSourceRanges:
   - 0.0.0.0/0
+# List of plugins to be install during Jenkins master start
+  InstallPlugins:
+      - kubernetes:0.11 
+      - workflow-aggregator:2.5
+      - credentials-binding:1.11
+      - git:3.2.0
```

Comment:
That's good to know. Agree with Lachie that this would be outside the context of this PR, but worth considering in another one.